torch/_torch_docs.py

```diff
@@ -1702,7 +1702,7 @@ def merge_dicts(*dicts):
 
 add_docstr(torch.unsafe_split,
            r"""
-unsafe_split(tensor, split_size_or_sections, dim=0) -> List of Tensors
+unsafe_split(tensor, split_size, dim=0) -> List of Tensors
```

Comment:
This seems like a dangerous change, because it would break backwards compatibility of the current PyTorch API.

For example, let's say I have some working code:
```
def forward(self, t: torch.Tensor) -> List[torch.Tensor]:
    return torch.split(t, split_size_or_sections=[2, 3, 5], dim=0)
```

Then, once this PR lands, my code will not work anymore. This is not a desirable behavior.

I think we should preserve backwards compatibility.